package.json

```diff
@@ -89,5 +90,8 @@
       "ethereumjs-util>ethereum-cryptography>keccak": true,
       "ethereumjs-util>ethereum-cryptography>secp256k1": true
     }
+  },
+  "resolutions": {
+    "@metamask/abi-utils": "portal:/Users/morten/Development/MetaMask/abi-utils"
```

Comment:
.

Reply:
This is temporary while waiting on a new release of `abi-utils`.